src/Umbraco.Cms.Api.Delivery/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -49,14 +52,19 @@ public static IUmbracoBuilder AddDeliveryApi(this IUmbracoBuilder builder)
                 ? provider.GetRequiredService<RequestContextOutputExpansionStrategy>()
                 : provider.GetRequiredService<RequestContextOutputExpansionStrategyV2>();
         });
+
         builder.Services.AddSingleton<IRequestCultureService, RequestCultureService>();
         builder.Services.AddSingleton<IRequestSegmmentService, RequestSegmentService>();
         builder.Services.AddSingleton<IRequestSegmentService, RequestSegmentService>();
         builder.Services.AddSingleton<IRequestRoutingService, RequestRoutingService>();
         builder.Services.AddSingleton<IRequestRedirectService, RequestRedirectService>();
         builder.Services.AddSingleton<IRequestPreviewService, RequestPreviewService>();
+
+        // Webooks register a more basic implementation, remove it.
+        builder.Services.RemoveAll(descriptor => descriptor.ServiceType == typeof(IOutputExpansionStrategyAccessor));
```

Comment:
Not checked, but not sure you need this.  Given you are registering a singleton for this interface on the next line, won't that automatically replace any previously registered implementation?